script/Ciel/Day 5/05_01_CIEL05_2_2.txt

```diff
@@ -821,7 +821,7 @@ His mouth opens wide, as if he's about to vomit up all of his internal organs.
 -- Page 359, Offset 11817.
 -- その、被害妄想じみた怨念が、
 // His hatred-grudge influenced-colored-fuelled by his delusion of persecution-paranoia-persecution complex
-His paranoia-tinged hatred...
+His paranoia-tinged hatred―――
```

Comment:
I'd put a lead-in on 11822 as well to make the connection clearer

Reply:
done